stylesheets/layout/footer.css

```diff
@@ -0,0 +1,80 @@
+/* footer css */
+
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: var(--secondary-400);
+  padding: 2rem 0;
+  gap: 1rem;
+  color: var(--white);
+  margin-top: auto;
+}
+
+.footer-head {
+  font-size: var(--font-s-base);
+}
+
+.footer-head-link {
+  font-size: var(--font-s-base);
+  font-weight: var(--font-w-medium);
+  color: var(--active-color);
+}
+
+.footer-head-link:hover {
+  color: var(--active-hover);
+}
+
+/* social media css  */
+.social-buttons {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  width: fit-content;
+  gap: 1rem;
+}
+
+.social-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;

```

Comment:
Same suggestion as above. Extract it into a utility class.

Reply:
Added